### DISCUSSION TITLE
Thermostat Models

### DISCUSSION THREAD

**Discussion by donavanbecker** [d0n4v4nb3ck3r/homebridge-honeywell-home#377] [General]
Just wanted to start the discussion to see what thermostat models people have used with the plugin, if you could share?

**Comment by keithkorg** [comment]
I have a t9 but I can't get the refresh token. how do I get that? I have added my secret and app consumer key. im not getting the link to account screen.

**Reply by donavanbecker** [reply]
Do you have homebridge UI?

**Reply by keithkorg** [reply]
I do now. I was running hoobs. Everything works great.

**Reply by donavanbecker** [reply]
> I do now. I was running hoobs. Everything works great. 

You switched from HOOBs to Homebridge with UI?

Yeah we have a built in Authentication built into the UI thanks to the developer of Homebridge UI.

**Comment by tebasham** [comment]
T5 - RCHT8612WF2005

**Comment by dkossman** [comment]
I have a couple of TH8321R1001 thermostats plus a remote temperature/humidity sensor, and the Redlink internet gateway.  Before i go through the setup process - anyone know whether this is supported by this plugin?

thanks
